apps/dashboard/src/components/contract-functions/interactive-abi-function.tsx

```diff
@@ -319,7 +324,7 @@ export const InteractiveAbiFunction: React.FC<InteractiveAbiFunctionProps> = ({
                 code={formatResponseData(data || readData)}
               />
               {typeof readData === "string" &&
-                readData?.startsWith("ipfs://") && (
+                (readData as string)?.startsWith("ipfs://") && (
```

Comment:
is this required? the `typeof` check above does not correctly infer this?

Reply:
yeah.. let me grab the error message

Reply:
Inferred type:
```
const readData: readonly unknown[] | undefined
```

Error:
```
Property 'startsWith' does not exist on type 'never'.ts(2339)
```